chut/__init__.py

```diff
@@ -233,6 +233,8 @@ def __init__(self, *args, **kwargs):
         self.kwargs = kwargs
         if 'sh' in kwargs:
             kwargs['shell'] = kwargs.pop('sh')
+        if kwargs.get('shell') and len(args) > 1:
+            raise TypeError("One argument at most is allowed when shell=True")
```

Comment:
This is because it doesn't make much sense to me to concatenate with a space possibly in the middle of an argument. For example

    do('something "great', a, b, 'of course"')

once parsed by the shell, `a` and `b` won't be individual args. They will be part of the string that begins with "great" and ends with "course".

But I guess this is too much complication and should just be allowed